tests/test_config/test_predicates.py

```diff
@@ -38,6 +39,19 @@ def test_ordering_xhr_and_request_method_trump_only_containment(self):
     def test_ordering_number_of_predicates(self):
         from pyramid.config.predicates import predvalseq
 
+        order0, _, _ = self._callFUT(
```

Comment:
I named this object 0 to avoid having to rename everyting and mess up the git blame on lines 139-148.

Reply:
seems good to me